### PR TITLE
Jmn/master

### PR DESCRIPTION
**Pull request by jennaplusplus**
Wave 1!

### CODE REVIEW

**Comment by kariabancroft** [comment]
Nice job with your incremental commits via git - history looks good

File: views/about.erb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
...AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQkIHSEJCQkJCQkJCQkI1QkJCRiRCQkI8Fk8lQkJCBTFCQkIvIUIyLEJCCx9CQj0tQkJCAT42CUJCQgQVABpDLkJCQkUmIBlCQkIMCClJJhBCQkJLEhJJDhIPNwpJEgJCQkJCFxISEhISEhISEhIwQkJCUCYSEhISEhISEhISNEJCQhsSEhJJSUkSEhISEydCQismEg0RKkw7A0pOTjhHQkAmEkMoQkJCQkJCQkIcHTlCBiIjQkJCQkJCQkJCQhhEQkIUQkJCQkJCQkJCQkJBTUJCQkJCQkJCQkJCQkJCMz9CQkJCQkJCQkJCQkJCHjpCQkJCQkJCQkJCQkJCQkJCQgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=" rel="icon" type="image/x-icon" />
```
Comment (kariabancroft) [inline views/about.erb:5]: Is this a URL?

File: views/blog.erb [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      <h3>Hello, World!</h3>
+      <h4>October 26, 2015</h4>
+    </header>
+    <p>
+      Blogging... I hate blogging... just going to post some song lyrics instead...
+    </p>
+
+    <h5><a href="https://www.youtube.com/watch?v=ymei5EA5tz0">Daft Punk - Around the World</a></h5>
+    <p>
+      Around the world, around the world
```
Comment (kariabancroft) [inline views/blog.erb:25]: I can't wait until you put this dang thing in a loop!

File: views/about.erb (hunk 1 above)
Comment (jennaplusplus) [inline views/about.erb:5]: Yeah, it's for the favicon... I couldn't get it to work when I just had the image/icon locally. :/

Comment (kariabancroft) [inline views/about.erb:5]: Hmmm... that's lame. I know Jennie was trying the same thing but couldn't get it to work as far as I know

**Pull request merged**